src/compass/s1_geocode_slc.py

```diff
@@ -121,8 +121,8 @@ def run(cfg: GeoRunConfig):
             ctype = h5py.h5t.py_create(np.complex64)
             ctype.commit(geo_burst_h5['/'].id, np.string_('complex64'))
 
-            grid_path = f'{root_path}/CSLC/grids'
-            grid_group = geo_burst_h5.require_group(grid_path)
+            data_path = '/CSLC/data'
+            grid_group = geo_burst_h5.require_group(data_path)
```

Comment:
```suggestion
            data_group = geo_burst_h5.require_group(DATA_PATH)
```
Remove use `grid`.
Add import up top to get `DATA_PATH` from `h5_helpers.py`.